### PR TITLE
Update how-to-create-and-edit-permission-levels.md

### PR DESCRIPTION
**Pull request by jwaite007**
Reach out to JWaite at Microsoft for more info if needed.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/how-to-create-and-edit-permission-levels.md [hunk 1]
```diff
@@ -110,6 +110,9 @@ You can delete any permission level, except for **Full Control** and **Limited A
   
 To delete a permission level, on the **Permissions Levels** page, select the check box next to the permission level that you want to delete, and then chose **Delete Selected Permission Levels**.
   
+> [!CAUTION]
+> It is possible to create permission levels which do not grant the minimum required permissions. The modern user experience uses client side rendering which requires that all users have the "Use Remote Interfaces" permission. Removing "Use Remote Permissions" from a permissions level will result in Access Denied errors that either prevent features on a modern page from loading or prevent the entire modern page from loading, resulting in an Access Denied error for the user.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/how-to-create-and-edit-permission-levels.md:114]:
```suggestion
> It is possible to create permission levels that do not grant the minimum required permissions. The user experience uses client-side rendering which requires that all users have the **Use Remote Interfaces** permission. Removing Use **Remote Permissions** from a permissions level will result in **Access Denied** errors that either prevent features on a page from loading or prevent the entire page from loading, resulting in an **Access Denied** error for the user. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jwaite007, please commit my recommended changes if you agree. For information, the term "modern" is now discouraged in Microsoft docs as it is jargon that means very little (except for "modern authentication", which is an officially blessed term). Many thanks.

**Comment by jwaite007** [comment]
> Copy review - @jwaite007, please commit my recommended changes if you agree. For information, the term "modern" is now discouraged in Microsoft docs as it is jargon that means very little (except for "modern authentication", which is an officially blessed term). Many thanks.

How do we differentiate that this issue only happens with our new user experience pages which use client side rendering without using the word modern? This isn't an issue with the classic pages and customers that have removed Use Remote Interfaces are now having problems as we push out new features which rely on client side rendering.

**Review by JohanFreelancer9 — APPROVED** [review]
@jwaite007 - Thank you for clarifying. I take your point and have deleted my suggested changes.

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**